plugins/experimental/uri_signing/uri_signing.c

```diff
@@ -218,6 +224,32 @@ TSRemapDoRemap(void *ih, TSHttpTxn txnp, TSRemapRequestInfo *rri)
       checkpoints[cpi++] = mark_timer(&t);
     }
     jws = get_jws_from_cookie(&client_cookie, &client_cookie_sz_ct, package);
+  } else {
+    /* There has been a JWS found in the url */
+    /* Strip the token from the URL for upstream if configured to do so */
+    if (config_strip_token((struct config *)ih)) {
+      if (strncmp(url, strip_uri, strip_ct) != 0) {
```

Comment:
Could this be simplified to checking `strip_ct != url_ct`? I don't think it's possible to remove a token without changing the length and that would be a much more efficient check.

Reply:
Yep good point. Changed it